website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -6,14 +6,22 @@ sidebar_label: "MetricFlow time spine"
 tags: [Metrics, Semantic Layer]
 ---
 
-It's common in analytics engineering to have a date dimension or "time spine" table as a base table for different types of time-based joins and aggregations. The structure of this table is typically a base column of daily or hourly dates, with additional columns for other time grains, like fiscal quarter, defined based on the base column. You can join other tables to the time spine on the base column to calculate metrics like revenue at a point in time, or to aggregate to a specific time grain.
+It's common in analytics engineering to have a date dimension or "time spine" table as a base table for different types of time-based joins and aggregations. The structure of this table is typically a base column of daily or hourly dates, with additional columns for other time grains, like fiscal quarters, defined based on the base column. You can join other tables to the time spine on the base column to calculate metrics like revenue at a point in time, or to aggregate to a specific time grain.
 
-MetricFlow requires you to define a time spine table as a project level configuration, which then is used for various time-based joins and aggregations, like cumulative metrics. At a minimum, you need to define a time spine table for a daily grain. You can optionally define a time spine table for a different granularity, like hourly.
+MetricFlow requires you to define a time spine table as a project-level configuration for time-based joins and aggregations, like cumulative metrics. 
```

Comment:
```suggestion
MetricFlow requires you to define a time spine table as a model-level configuration in the Semantic Layer for time-based joins and aggregations, such as cumulative metrics. This configuration informs dbt which model should be used for time range joins. It is especially useful for cumulative metrics or calculating time-based offsets.
```

Reply:
Yea this is more clear. I would update this to sentence to:

> It is especially useful for cumulative metrics or calculating time-based offsets.
The time-spine model is joined to other tables when calculating certain types of metrics or dimensions. MetricFlow will join to the time-spine model in the compiled SQL for the following types of metrics and dimensions:
* Cumulative metrics
* Metric offsets
* Conversion metrics
* Slowly Changing Dimensions
* Metrics with the join_to_timespine configuration set to true
To see the generated SQL for the metric and dimension types that utilize time spine joins refer to the docs, or add the `compile=True` flag when querying the semantic layer to return the compiled SQL.